csrc/fusion.cpp

```diff
@@ -259,26 +259,28 @@ void Fusion::addInput(Val* input) {
   all_tv_uses_valid_ = false;
 }
 
-void Fusion::addOutput(Val* output) {
-  // We currently don't support explicitly outputing aliased inputs. This is
-  // because they are already marked as output for in-place update. It's tricky
-  // to allow marking them explicitly as real output, since that requires us to
-  // register/identify output not only by `Val*` pointer, but also by indices;
-  // it also requires us to magically arrange `outputs_` entries in proper order
-  // ^^^ this doesn't look intuitive on `outputs_` in fusion.
-  // I think we can solve this by marking addOutput on io_alias_ keys after
-  // fusion is fully defined. Tracking this in #1488
```

Comment:
#1488 doesn't seem to be related to this.

Reply:
Can you explain how the concerns in this comment are addressed:
```
  // since that requires us to
  // register/identify output not only by `Val*` pointer, but also by indices;
  // it also requires us to magically arrange `outputs_` entries in proper order
  // ^^^ this doesn't look intuitive on `outputs_` in fusion.
```
I do not have the context behind re-arranging outputs.

Reply:
I believe it refers to https://github.com/csarofeen/pytorch/issues/1488

Reply:
Ha good catch @wujingyue totally forgot about that repo. :face_exhaling:

Reply:
Sorry @Priya2698 , I pinged you here because #1488 was under your name.
Updated the PR description.